### PR TITLE
Release 0.5

### PR DESCRIPTION
**Pull request by Lukeslux**

### CODE REVIEW

File: doc/manual/design/architecture.md [hunk 1]
```diff
@@ -23,24 +23,24 @@ Using these tools, we will were able to define our system as follows:
 
 ![Overview Diagram](asset/overview.png)
 
-The Flask server defines two endpoints accessable if the session key is stored in a cookie. If this session is not found, the server redirects the user to the Zeeguu login page. 
+The Flask server defines two endpoints accessible if the session key is stored in a cookie. If this session is not found, the server redirects the user to the Zeeguu login page. 
 
-The root endpoint simply delivers the articles.html page on a valid GET request. This page defines all styles and structure of the articles listing and the subscription menu, but all functionality is implemented by the package which it invokes. The subscription package make use of the Cache class in order to store article listings locally, but Caching is not specific to this package.
+The root endpoint simply delivers the articles.html page on a valid GET request. This page defines all styles and structure of the articles listing and the subscription menu, while all the functionality is implemented by the package which it invokes. The subscription package makes use of the Cache class in order to store article listings locally, but Caching is not specific to this package.
```
Comment (mircealungu) [inline doc/manual/design/architecture.md]: drop the simply. not necesary

File: src/umr/static/scripts/app/translation/main.js [hunk 2]
```diff
@@ -82,11 +70,36 @@ function disableSelection() {
...
+    $(config.HTML_ID_TOGGLETRANSLATE).removeClass('mdl-button--disabled');
+}
+
+/* Attach Zeeguu tag click listener. */
+function attachZeeguuListeners () {
+    /* When a translatable word has been clicked,
+     * either try to translate it, speak it, or open an alternative
+     * translation window.  */
+    $(config.HTML_ZEEGUUTAG).click(function(event) {
+        if (!$(config.HTML_ID_TOGGLETRANSLATE).hasClass('mdl-button--disabled'))
```
Comment (mircealungu) [inline src/umr/static/scripts/app/translation/main.js]: this double negation is hard to parse. 
it seems that translations don't happen "if not toggle_translate is disabled". 
is there any renaming that you can think about that would make this more readable?

File: src/umr/static/scripts/app/translation/main.js [hunk 3]
```diff
@@ -82,11 +70,36 @@ function disableSelection() {
...
+            return;
+        var target = $(event.target);
+        if ( target.is(config.HTML_ZEEGUUTAG) ) {
+            if (!translator.isTranslated(this)) {
+                translator.translate(this);
+            }
+        } else if (target.is(config.HTML_ORIGINAL) ) {
+            speaker.speak($(this).find(config.HTML_ORIGINAL).text(), FROM_LANGUAGE);
+        } else if (target.is(config.HTML_TRANSLATED) ) {
+            alterMenu.constructAndOpen(this.children[1]);
```
Comment (mircealungu) [inline src/umr/static/scripts/app/translation/main.js]: what is this here? 
can you add a comment for my sake? :)

File: webpack.config.js [hunk 4]
```diff
@@ -6,11 +6,11 @@ var inProduction = process.env.NODE_ENV === 'production';
...
     entry: {
-		subscription: './src/static/scripts/app/subscription/main.js',
-		translation:  './src/static/scripts/app/translation/main.js'
+		subscription: './src/umr/static/scripts/app/subscription/main.js',
+		translation:  './src/umr/static/scripts/app/translation/main.js'
     },
     output: {
-        path: path.join(__dirname, './src/static/scripts/dist'),
+        path: path.join(__dirname, './src/umr/static/scripts/dist'),
         filename: '[name].entry.js'
```
Comment (mircealungu) [inline webpack.config.js]: hmm. silly question. why is there an .entry in the name of the output file?

Comment (Lukeslux) [inline webpack.config.js]: Not necessary, just a naming convention. Can be simplified to [name].js.

**Comment by Lukeslux** [comment]
Should be fixed now :+1:
